### DISCUSSION TITLE
Not able to log SQL Query using 1.8.0 version

### DISCUSSION THREAD

**Discussion by debraj-manna** [gavlyukovskiy/spring-boot-data-source-decorator#82] [General]
I am new to sprint-boot-data-source decorator. I am trying to log all SQL Query in console.

I have added `p6spy-spring-boot-starter` in my pom.xml.

```
NFO] |  +- org.springframework:spring-tx:jar:5.3.25:compile
[INFO] |  +- com.github.gavlyukovskiy:p6spy-spring-boot-starter:jar:1.8.0:compile
[INFO] |  |  +- com.github.gavlyukovskiy:datasource-decorator-spring-boot-autoconfigure:jar:1.8.0:compile
[INFO] |  |  \- p6spy:p6spy:jar:3.9.0:compile
[INFO] |  \- org.springframework.boot:spring-boot-starter-jooq:jar:2.7.9:compile
[INFO] |     \- jakarta.activation:jakarta.activation-api:jar:1.2.2:compile
```

In my application.properties I have also added

```
decorator.datasource.p6spy.enable-logging=true
decorator.datasource.p6spy.multiline=true
decorator.datasource.p6spy.logging=sysout
```

But I am still not seeing the SQL query being logged. Can someone let me know what is going wrong?

**Comment by debraj-manna** [comment]
The same has been asked in [stackoverflow](https://stackoverflow.com/questions/75810022/log-sql-query-using-p6spy-and-spring-boot-2-7) also.

**Comment by gavlyukovskiy** [comment]
It's a bit hard for me to answer without any sample project. Can you please provide a sample project or check the differences between your project and [p6spy-sample](https://github.com/gavlyukovskiy/spring-boot-data-source-decorator/tree/master/samples/p6spy-sample)?
